snapshotter/pkg/losetup/losetup_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package losetup
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/docker/go-units"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestLosetup(t *testing.T) {
```

Comment:
These are good happy path test cases but I think we could add some non-happy path case like:

* Removing/detaching a device that doesn't exist
* Attaching a device from a non-existent (or malformed?) image path
* Finding devices that don't exit

Reply:
Done